services/account-without-juggler/repositories/account/datasources/mysqlconn.ts

```diff
@@ -0,0 +1,201 @@
+const debug = require("debug")("account-without-juggler");
+const mysql = require("mysql");
+const db = require("mysql-promise")();
+import { CrudConnector } from "loopback-next/packages/repository/src/crud-connector";
+import { IError } from "@types/mysql";
+import {
+  Class,
+  DataSource,
+  Entity,
+  EntityData,
+  Filter,
+  ObjectType,
+  Options,
+  Where
+} from "loopback-next/packages/repository";
+
+export class MySqlConn implements CrudConnector {
+  private connection: any;
```

Comment:
We don't have that exported in Repository, but I was trying to make it of type `IConnection` from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mysql/index.d.ts#L39-#L83, but the return types for `connect` and `disconnect` don't align with ours (`Promise<void>`)